packages/manager/apps/web/client/app/hosting/multisite/logs/hosting-multisite-logs.controller.js

```diff
@@ -67,12 +67,7 @@ angular.module('App').controller(
           }
         })
         .catch(() => {
-          this.Alerter.error(
-            this.$translate.instant(
-              'hosting_tab_DOMAINS_multisite_logs_generation_error',
```

Comment:
Maybe we can get remove translations for the given key `hosting_tab_DOMAINS_multisite_logs_generation_error`.

Reply:
This is done. Thanks a lot :)